test/Makefile

```diff
@@ -9,7 +9,21 @@ ATOMIC_WGET := $(CUR_DIR)/../tools/atomic_wget.sh
 BUILD_DIR := $(CUR_DIR)/build
 INITRAMFS := $(BUILD_DIR)/initramfs
 INITRAMFS_FILELIST := $(BUILD_DIR)/initramfs.filelist
-INITRAMFS_IMAGE := $(BUILD_DIR)/initramfs.cpio.gz
+# Supported encodings: `gzip` and `none`
+INITRAMFS_ENCODING ?= gzip
```

Comment:
I think the word "encoding" is a little bit confusing. The "cpio" format is also a kind of encoding. But this parameter does not affect whether "cpio" is used or not. I don't think we have any plan to support any other compression encoders. So we can simply name it `INITRAMFS_SKIP_GZIP`.